core/designsystem/src/main/java/com/google/samples/apps/nowinandroid/core/designsystem/theme/Theme.kt

```diff
@@ -173,55 +174,63 @@ val DarkAndroidBackgroundTheme = BackgroundTheme(color = Color.Black)
 /**
  * Now in Android theme.
  *
- * The order of precedence for the color scheme is: Dynamic color > Android theme > Default theme.
- * Dark theme is independent as all the aforementioned color schemes have light and dark versions.
- * The default theme color scheme is used by default.
- *
  * @param darkTheme Whether the theme should use a dark color scheme (follows system by default).
- * @param dynamicColor Whether the theme should use a dynamic color scheme (Android 12+ only).
- * @param androidTheme Whether the theme should use the Android theme color scheme.
+ * @param androidTheme Whether the theme should use the Android theme color scheme instead of the
+ *        default theme. If this is `false`, then dynamic theming will be used when supported.
  */
 @Composable
 fun NiaTheme(
     darkTheme: Boolean = isSystemInDarkTheme(),
-    dynamicColor: Boolean = false,
     androidTheme: Boolean = false,
-    content: @Composable() () -> Unit
+    content: @Composable () -> Unit
+) = NiaTheme(
+    darkTheme = darkTheme,
+    androidTheme = androidTheme,
+    disableDynamicTheming = false,
+    content = content
+)
+
+/**
+ * Now in Android theme. This is an internal only version, to allow disabling dynamic theming
```

Comment:
It should live in the test srcSet then

Reply:
We also use it in the previews for `Background`